pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -93,6 +88,4 @@ stdenv.mkDerivation {
 
     runHook postInstall
   '';
-
-  inherit (unwrapped) meta;
```

Comment:
@abbradar @groxxda there was already a `inherit unwrapped;` line, doesn't that mean that the `meta` attribute is already inherited?